docs/docs/non-cloud-deployment.md

```diff
@@ -0,0 +1,114 @@
+
+# Non-cloud deployment
+
+Conclave recommends users to [deploy applications in Azure VMs](machine-setup.md) that support the latest 
```

Comment:
I would remove `that support the latest attestation protocols.` Maybe we can clarify that we want to use them to use Azure because things are easier.

Reply:
Done.